controllers/users/create/swagger.js

```diff
@@ -0,0 +1,51 @@
+export const createUser = {
+  paths: {
+    '/users': {
+      post: {
```

Comment:
I'm not sure we have POST: /users described in `Endpoints.txt`

Reply:
Maybe it's for creating users by admin.

Reply:
Admin create another admin:)

Reply:
OK, already removed.